src/components/other/Avatar.tsx

```diff
@@ -0,0 +1,23 @@
+import { ThemeSize } from "@src/theme";
+import { observer } from "mobx-react-lite";
+
+
+interface IAvatarProps {
+    source?: string
+    size?: ThemeSize
+    onClick?: () => void
+    className?: string
+}
+
+export const Avatar: React.FC<IAvatarProps> = observer(({ source, size, onClick, className}) => {
+    return (
+        <img src={source} 
+        onClick={onClick} 
+        alt="avatar-photo" 
+        className={`${className} ${size}`}/>
```

Comment:
tukaj dodaš samo "l" classname. Z size poveš avatarju kakšen width in height naj ima. ThemeDefault ne naredi "l" classname pač pa recimo "width-l" ali "height-l".

Daj inspect element na stran, idi pod network, reloadaj stran in poglej main.js request. Boš videl v štartu veliko CSSja. To kar vidiš zaenkrat večinoma ThemeDefault.scss zgenerira. Boš videl tudi.

Reply:
Se pravi dodam samo "className" in določim avatar class(izgled po defaultu). Dodam noter pod className tudi size? Ker jaz si zamišlam,  da size potem spreminjam vsakemu avatarju, vendar more Avatar komponenta nekako vedeti kakšen je width in height?

Reply:
Izgled in size nemores zdruzit. Lahko pa das 2 classname-a. En za izgled en za size. 

V default props si pa dolocil da je po defaultu "l".

Torej tvoji classname zgleda pol tak:

avatar width-${size} height-${size} ${className}

Pa poglej si zakaj width-size deluje sploh tak kot sem opisal, da inspectas default css

Reply:
Za vsaki dinamicen property bos moral dati svoj classname. Imas default style + dinamicen style

Reply:
Ajaaa... sem pogledal v ThemeDefault.scss. Jaz sem mu dodeljeval samo value brez atributa, če prav razumem? Bi mogel verjetno upoštevati $size-attributes?

Reply:
Tako. Tukaj se ustvarijo default classnames glede na atribut in property.

Torej ce mas atribut width in size l, se ustvari globalen classname "width-l".

Reply:
Okej, zaj pa mi je jasno :). Bom še porihtal pa pošljem.... se pravi zdaj moja avatar komponenta sprejme "size", 
"source", kasneje pa še onClick, ki preusmeri na profil page predvidevam ter po potrebi "custom" className?

Reply:
Tako.

Ampak pazi. Required je samo source, ostalo je vse optional.

Sourca mora biti drugace avatarja nemores prikazat
Onclick je optional, lahko je lahko ni
Size je optional, ce size ni specificiran je po defaultu l
Classname je optional

Reply:
Okej, razumljivo. Dodam samo po potrebi, ne? Hvala za enkrat, si morem še malo Sass pogledat... :) Porihtam to pa pošljem.

Reply:
Tako. Poglej si sass in css. Zdaj boma dosti to delala samo, da malo osvojis.

Optional je vedno po potrebi. Tudi ce je size po default "l". Ko uporabis komponento nerabis se enkrat specificirat l.

Reply:
sem to spremenil zdaj, pošljem PR.